covid_xprize/examples/prescriptors/neat/README.md

```diff
@@ -19,29 +19,28 @@ that scales quadratically with the number of calls.
 
 ### Setup
 
-These assume the `covid-xprize` repo is located at `<workspace>/covid-xprize`.
+Assuming you have cloned this repository and your working directory is currently the root of the cloned
```

Comment:
The README no longer assumes they've cloned the repo to any particular place or with any particular name. Everything is just relative to the root of wherever they cloned it.